### DISCUSSION TITLE
Chip-seq single cell

### DISCUSSION THREAD

**Discussion by A-legac45** [GreenleafLab/ArchR#1654] [Questions / Documentation]
Hello everybody, 

I am trying to use ArchR to analyse a dataset of sc chip-seq for specific histone marks. Is that possible?  Does other tools can be used ? 

I am encountering the two following problems with ARCHR : 

**1. If yes I do not get why if I put by default like this the code I get different clusters** 

proj1 <- addIterativeLSI(ArchRProj = proj1, useMatrix = "TileMatrix", name = "IterativeLSI3")

proj1 <- addClusters(input = proj1, reducedDims = "IterativeLSI3", force = TRUE)

**2. and If I look to the parameters and try to improve I get only one cluster**

proj1 <- addIterativeLSI(
  ArchRProj = proj1,
  useMatrix = "TileMatrix", 
  name = "IterativeLSI", 
  iterations = 2, 
  clusterParams = list( #See Seurat::FindClusters
    resolution = c(0.2), 
    sampleCells = 10000, 
    n.start = 10
  ), 
  varFeatures = 25000, 
  dimsToUse = 1:30, force = TRUE
)

proj1 <- addIterativeLSI(
  ArchRProj = proj1,
  useMatrix = "TileMatrix", 
  name = "IterativeLSI2", 
  iterations = 2, 
  clusterParams = list( #See Seurat::FindClusters
    resolution = c(0.1, 0.2, 0.3, 0.4, 0.5, 2)
  ), 
  varFeatures = 20000, 
  dimsToUse = 1:30, force = TRUE
)

proj1 <- addClusters(input = proj1, reducedDims = "IterativeLSI", method = "Seurat",
                    name = "Clusters", resolution = 0.8, force = TRUE)

**Comment by rcorces** [comment]
sc ChIP-seq is not a supported use case for ArchR and we dont have bandwidth to help troubleshoot related errors. Sorry to not be able to help on this.
